prow/cmd/branchprotector/protect_test.go

```diff
@@ -27,6 +27,103 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
+func TestJobRequirements(t *testing.T) {
```

Comment:
Might be easier (for me, anyway) to understand if `AlwaysRun` and `SkipReport` were always explicitly called out -- hard to keep track and brain-default them.

Reply:
Done